### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1072] [Galaxy Trivia] [answered]
```
2 15 54 171 522 ?
```

**Comment by ParanoidUser** [comment]
@JavaScriptonit / @danieljuliusstein / @markaceee, any ideas? 🫣 #brainstorm

**Reply by JavaScriptonit** [reply]
Based on the given numerical sequence "2 15 54 171 522 ?", I can suggest the next number: 1575.

By applying a simple rule of multiplying the previous number by 3 and adding 3, we can obtain the following numbers: 2 * 3 + 3 = 9 9 * 3 + 3 = 30 30 * 3 + 3 = 93 93 * 3 + 3 = 282 282 * 3 + 3 = 849 849 * 3 + 3 = 2550

Therefore, the next number in the sequence can be 1575. However, this is just a conjecture and may not correspond to the actual rule or context, if any.

**Comment by NomanGhiasy** [comment]
1575

**Reply by JavaScriptonit** [reply]
The same for me!

**Comment by CelioAmaral** [comment]
2 15 54 171 522 ?

2 + 3 = 5

5 x 3 = 15

15 + 3 = 18

18 x 3 = 54

54 + 3 = 57

57 x 3 = 171

171 + 3 = 174

174 x 3 = 522

522 + 3 = 525

525 x 3 = 1575

This answer is: 1575

**Reply by JavaScriptonit** [reply]
the same for me!

**Comment by thuongtruong1009** [comment, marked as answer]
To find the pattern in the sequence, let's look at the differences between consecutive numbers:

15 - 2 = 13
54 - 15 = 39
171 - 54 = 117
522 - 171 = 351

We can see that the differences are increasing with each step. Specifically, the differences seem to be multiples of 3: 13 * 3 = 39, 39 * 3 = 117, 117 * 3 = 351.

So, to find the next number in the sequence, we can continue the pattern:

522 + 351 * 3 = 522 + 1053 = 1575

The next number in the sequence is 1575.

**Reply by JavaScriptonit** [reply]
same

**Comment by JavaScriptonit** [comment]
> ```
> 2 15 54 171 522 ?
> ```

Thank you for your trust! Based on the given numerical sequence "2 15 54 171 522 ?", I can suggest the next number: 1575.

By applying a simple rule of multiplying the previous number by 3 and adding 3, we can obtain the following numbers: 2 * 3 + 3 = 9 9 * 3 + 3 = 30 30 * 3 + 3 = 93 93 * 3 + 3 = 282 282 * 3 + 3 = 849 849 * 3 + 3 = 2550

Therefore, the next number in the sequence can be 1575. However, this is just a conjecture and may not correspond to the actual rule or context, if any.
